src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1817,15 +1829,10 @@ fd_exec_vm_syscall_test_run( fd_exec_instr_test_runner_t * runner,
   effects->frame_count = vm->frame_cnt;
 
   fd_log_collector_t * log = &vm->instr_ctx->txn_ctx->log_collector;
-  ulong logs_len = fd_log_collector_debug_len( log );
-  if( logs_len ) {
-    FD_TEST_CUSTOM( logs_len==1, "We only support syscalls tests with at most 1 log msg" );
-    uchar const * msg; ulong msg_sz;
-    fd_log_collector_debug_get( log, 0, &msg, &msg_sz );
+  if( log->buf_sz ) {
     effects->log = FD_SCRATCH_ALLOC_APPEND(
-      l, alignof(uchar), PB_BYTES_ARRAY_T_ALLOCSIZE( msg_sz ) );
-    effects->log->size = (uint)msg_sz;
-    fd_memcpy( effects->log->bytes, msg, msg_sz );
+      l, alignof(uchar), PB_BYTES_ARRAY_T_ALLOCSIZE( log->buf_sz ) );
+    effects->log->size = (uint)fd_log_collector_debug_sprintf( log, (char *)effects->log->bytes, 0 )-1;
```

Comment:
why -1?

Reply:
fixed (fd_log_collector_debug_sprintf used to return strlen+1, including the final \0)